src/extension/tab/helpers.ts

```diff
@@ -29,30 +28,39 @@ export function getQueries(
 
       queries.push({
         document,
-        source: document?.loc?.source,
```

Comment:
This is not used when [getting the query string](https://github.com/apollographql/apollo-client-devtools/pull/1052/files#diff-e846862c9de29c06c715196483bc3b28ed25a6741f2b846549112dd97128ff0eR133).